custom_components/healthchecksio/__init__.py

```
@@ -94,3 +98,96 @@ async def async_unload_entry(
+            return False
+        version = 2
+
+    _LOGGER.info("Migration to version %s successful", version)
+    return True
+
+
+def _migrate_1_to_2(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    entity_registry = er.async_get(hass)
+    data = dict(config_entry.data)
+
+    CONF_CHECK = "check"
```

Pylint (3.3.7)

[convention] Variable name "CONF_CHECK" doesn't conform to snake_case naming style

(C0103)